Update/ura_001.txt

```diff
@@ -432,7 +432,7 @@ void main()
 //rもっとも、それこそが二人の仲睦まじさの裏返しであり、多少嫉妬深いところも深い愛情があったからだろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "もっとも、それこそが二人の仲睦まじさの裏返しであり、多少嫉妬深いところも深い愛情があったからだろう。",
-			NULL, "Being on the flip side of their close relationship, I was a little jealous of their deep affection.", Line_Normal);
+			NULL, "However, that was the flip side of their close relationship, and she probably got jealous exactly because of how much she loved him.", Line_Normal);
```

Comment:
```suggestion
			NULL, "However, that was the flip side of their close relationship, and she probably got jealous precisely because of how much she loved him.", Line_Normal);
```